xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -739,8 +739,11 @@
  * NOTE: This macro uses variables defined in livetable_filterProperty . It was not meant to be used alone.
  *#
 #macro (livetable_filterStringListProperty)
```

Comment:
It's true we could simply add the script on a more global level and apply it to `select[multiple="multiple"]`, since bootstrap-select is only a layer on top of the existing controls and shouldn't break anything, but then I think it should probably be discussed.